### PR TITLE
Feature | Updated "hire us" content

### PR DESCRIPTION
**Pull request by monsteronfire**
This PR adds:
1. More content sections to the `hire us` page

### CODE REVIEW

File: _includes/hire_us/new_projects.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <div class="row justify-content-center mb-5">
+            <div class="col-md-7 text-center">
+                <h3 class="section-subheading text-muted">For projects where we staff all developer roles, we like to work as follows:</h3>
+            </div>
+        </div>
+        <div class="row text-center">
+            <div class="col">
+                <h4 class="service-heading">Scrum</h4>
+                <p class="text-muted">We carry out our work in two-week Sprints. We ensure that four key roles are
+                    present on the team: Product Owner, Scrum Master, Developers and QA Engineer(s).</p>
```
Comment (carloslima) [inline _includes/hire_us/new_projects.html]: s/present/covered/ ?
The way QA Eng is worded might send a different message than what was intended.

Comment (monsteronfire) [reply _includes/hire_us/new_projects.html]: Will leave this feedback for @calumhalcrow, as I just copy/pasted his content from "custom software development" service.

**Comment by monsteronfire** [comment]
@calumhalcrow I've made some changes to the copy. Please take a look when you're able to. Thanks!

**Comment by calumhalcrow** [comment]
Looks good! Merging...

**Pull request merged**